internal/transformer/transformer.go

```diff
@@ -329,6 +329,7 @@ func walkForServices(inputPath string, ts map[string]Transformer, bservices map[
 	knownProjectPaths := []string{}
 
 	err = filepath.Walk(inputPath, func(path string, info os.FileInfo, err error) error {
+		common.PlanProgressNumFiles++
```

Comment:
We might want to use atomic for this. Otherwise we might end up with panic. https://gobyexample.com/atomic-counters